app/javascript/packs/containers/NavigationBar/DropdownMenuContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+class DropdownMenuContainer extends React.Component {
+
+    buildingChange(event) {
+        this.props.history.push(`/buildings/${event.target.value}`)
+    }
+
+    render() {
+        const buildings = this.props.buildings;
+        const currentBuilding = this.props.currentBuilding;
+        return (<div>
+            <h1> {currentBuilding ? currentBuilding.name : null} </h1>
+            <select className={"dropdown-container"}
+                    onChange={this.buildingChange.bind(this)} value = {currentBuilding ? currentBuilding.id : "Choose a Building"}>
+                <option disabled value = {"Choose a Building"}> {"Choose a Building"}</option>
+                {Object.keys(buildings).map(id => {
+                    return (
+                        <option key = {id} value={id} > {buildings[id].name} </option>
```

Comment:
Style nit (sorry): can you remove the whitespace between your `=` and the angle brackets `<>`?